multiplexer.go

```diff
@@ -88,6 +95,8 @@ func (m *multiplexer) callback(topic string, data map[string]interface{}) {
 // If any of the topic matchers added for the handlers match the topic, the
 // multiplexer matches.
 func (m *multiplexer) match(topic string) bool {
+	// Here we explicitly don't make a copy of the outputs as the match
```

Comment:
Since the matcher is an arbitrary callback this is still leaving it open to deadlock. But I guess it would be much more perverse to try to interact with the multiplexer in a match call.

Reply:
If it is a problem, it is the caller's problem.